package/MDAnalysis/analysis/pca.py

```diff
@@ -294,19 +294,19 @@ def transform(self, atomgroup, n_components=None, start=None, stop=None,
 
         Parameters
         ----------
-        atomgroup: MDAnalysis atomgroup/ Universe
+        atomgroup : MDAnalysis atomgroup/ Universe
             The atomgroup or universe containing atoms to be PCA transformed.
-        n_components: int, optional
-            The number of components to be projected onto, Default none: maps
+        n_components : int, optional
+            The number of components to be projected onto, Default None: maps
             onto all components.
-        start: int, optional
+        start : int, optional
             The frame to start on for the PCA transform. Default: None becomes
             0, the first frame index.
-        stop: int, optional
+        stop : int, optional
```

Comment:
rewrite, see above for example. Also replace n_frames with explicit "the total number of frames in the trajectory".

Reply:
Done